pkg/schedule/checker/merge_checker.go

```diff
@@ -88,13 +90,16 @@ func (m *MergeChecker) Check(region *core.RegionInfo) []*operator.Operator {
 		return nil
 	}
 
+	// update the split cache.
+	// It must be called before the following merge checker logic.
+	m.splitCache.UpdateTTL(m.conf.GetSplitMergeInterval())
```

Comment:
BTW, will it have a risk? If the heartbeat is not send in time?

Reply:
At the same time, we wait until 90% of the heartbeats are collected before checking.

Reply:
There still be 10% and merge check may create lots of scheduling?

Reply:
The behavior is similar in this case. For example, if split-merge-interval is set to 1s.
In the past, it would pass the recently started check and then run `UpdateTTL`
Now, it will run `UpdateTTL` first and then pass the recently started check.

Perhaps we can add a longer time before recently started check, such as `expireTime := m.startTime.Add(m.conf.GetSplitMergeInterval()+5*time.Minute)`